controllers/selfnoderemediation_controller.go

```diff
@@ -931,3 +936,13 @@ func IsOwnedByNHC(snr *v1alpha1.SelfNodeRemediation) bool {
 	}
 	return false
 }
+
+// getNodeName checks for the node name in snr's commonAnnotations.NodeNameAnnotation. If it does not exist it assumes the node name equals to SNR CR's name and returns it.
```

Comment:
uppercase SNR? Or maybe CR instead? Maybe use the actual annotation's key value in the comment? 🤔 

just curious, AI generated? 😉

Reply:
Nope :grin: 
https://github.com/medik8s/fence-agents-remediation/blob/83e25ea071740e4a768a5f7eccc7e8140450297c/controllers/fenceagentsremediation_controller.go#L328-L338